src/messages/datastore/latestMessages.js

```diff
@@ -172,8 +172,8 @@ export default {
       // needed for websocket updates
       if (!conversations) return
       for (const conversation of conversations) {
-        if (conversation.type === 'pickup') {
-          dispatch('pickups/maybeFetch', conversation.targetId, { root: true })
+        if (conversation.type === 'activity') {
```

Comment:
(same as before) Don't change yet, conversation type is part of API (backend sends it to us)

Reply:
Fixed!